src/css/index.css

```diff
@@ -0,0 +1,535 @@
+}
+
+.banner__content-item:nth-child(3) .banner__content-background {
+    opacity: 40%;
+}
+
+.banner__content-item:nth-child(4) .banner__content-background {
+    opacity: 10%;
+}
+
+.banner__content-item:nth-child(4) .banner__content-text {
+    background-color: var(--button-bg-color);
+    color: white;
+}
+
+.banner__content-text {
+    position: absolute;
+    padding: 10px;
+    background-color: var(--banner-bg-color);
+    font-size: var(--h3-font);
+    word-wrap: break-spaces;
+}
+
+/*##title*/
+
+.title {
+    margin-top: var(--top-margin);
+    border-bottom: 1px solid var(--title-border-color);
+}
+
+.title__content {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.title__content-text {
+    width: 80%;
+}
+
+.title__content-main-text {
+    font-size: var(--h1-font);
+    color: var(--main-text-color);
+}
+
+.title__content-subtext {
+    font-size: var(--h3-font);
+    color: var(--sub-text-color);
+}
+
+.title__content-img {
+    font-size: var(--h2-font);
+    color: var(--button-fg-color);
+}
+
+.title__content-button {
+    margin-bottom: 65px;
+    width: 20%;
+    padding: 15px;
+    font-size: var(--h2-font);
+}
+
+/*##row*/
+
+.row {
+    margin-top: var(--top-margin);
+}
+
+.row__content {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    width: 100%;
+}
+
+.row__content-item {
+    display: flex;
+    justify-content: left;
+    align-items: center;
+    width: 50%;
+    margin: 10px auto;
+}
+
+/*rounded images*/
+.row__content-span {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 2px solid var(--row-circle-color);
+    border-radius: 130px;
+    min-height: 80px;
+    min-width: 80px;
+    width: 8vw;
+    height: 8vw;
+    text-align: center;
+}
+
+.row__content-img {
+    display: inline;
+    font-size: 40px;
+    color: var(--content-img-color);
+}
+
+.row__content-article {
+    margin: 5%;
+}
+
+.row__content-title {
+    font-size: var(--h2-font);
+    color: var(--main-text-color);
+}
+
+.row__content-text {
+    margin-top: 10px;
+    font-family: var(--sub-font);
+    font-size: var(--p-link-font);
+    color: var(--sub-text-color);
+}
+
+/*##subscribe*/
+
+.subscribe {
+    margin-top: var(--top-margin);
+}
+
+.subscribe__content {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    border-radius: 5px;
+    height: 100px;
+    background: var(--subscribe-bg-color);
+}
+
+.subscribe__content-text {
+    width: 25%;
+    text-align: center;
+    font-size: var(--h1-font);
+    color: var(--subscribe-title-color);
+}
+
+.subscribe__content-form {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    margin-right: 10px;
+    width: 70%;
+}
+
+.subscribe__content-input {
+    border: none;
+    border-radius: 2px;
+    width: 33%;
+    padding: 10px 15px;
+    font-family: var(--sub-font);
+    font-size: var(--p-link-font);
+}
+
+.subscribe__content-button {
+    width: 20%;
```

Comment:
what if the text is bigger?
<img width="608" alt="Screen Shot 2020-03-30 at 1 23 19 PM" src="https://user-images.githubusercontent.com/2150198/77902421-cf2adc00-7289-11ea-9ae8-615cc870a25b.png">

adjust for all the buttons on the page.

also, the text of the button is better to have in one line(by default) and cut if there is no
space.